lbrynet/conf.py

```diff
@@ -583,6 +584,9 @@ def get_db_revision_filename(self):
         return os.path.join(self.ensure_data_dir(), self['DB_REVISION_FILE_NAME'])
 
     def get_conf_filename(self):
+        """
+        Return local configuration filename. If file does not exist, return None.
+        """
```

Comment:
Instead of the comment you added I'd like to rename the method name to something like: `get_valid_settings_filename`. This will explicitly document everywhere that the settings filename returned will be validated (at least that it exists). This is especially important since typically a method named similar to `get_conf_filename` would appear to be without performance penalty but in this case you are actually accessing the file system and checking that the file exists... every time it's called; the method name should reflect this or standout somehow.